specs/phase0/fork-choice.md

```diff
@@ -174,6 +175,13 @@ def get_current_slot(store: Store) -> Slot:
     return Slot(GENESIS_SLOT + get_slots_since_genesis(store))
 ```
 
+#### `get_current_store_epoch`
+
+```python
+def get_current_store_epoch(store: Store) -> Slot:
```

Comment:
```suggestion
def get_current_epoch(store: Store) -> Slot:
```

We already have `get_current_slot` querying over a store, seems natural to have these two names aligned

Reply:
That would be my preference too - but that would collide with [`get_current_epoch(state: BeaconState) -> Epoch`](https://github.com/ethereum/consensus-specs/blob/468b5be7b8a516d6fbc60ad8abec8830a5181bf4/specs/phase0/beacon-chain.md#get_current_epoch) in the state transition spec.